A1.md

```diff
@@ -3,14 +3,14 @@ HTTP CONNECT Proxy Support
 * Author(s): Mark D. Roth (roth@google.com), Eric Anderson (ejona@google.com)
 * Approver: a11r
 * Status: Draft
-* Implemented in: (C-core and Java are in progress)
-* Last updated: 2017-01-18
+* Implemented in: C-core (Java is in progress)
+* Last updated: 2017-01-25
 * Discussion at: https://groups.google.com/d/topic/grpc-io/zG7iYpDw6lk/discussion
 
 ## Abstract
 
 Describes what types of TCP-level proxy setups gRPC will support and
-how those setups interact with load balancing.
+how those setups interact with per-call load balancing.
```

Comment:
I've added the term "client-side", but added a parenthetical to make it clear that this includes both grpclb and round_robin.